pkg/accounting/accounting.go

```diff
@@ -177,9 +182,13 @@ func (a *Accounting) Balance(peer swarm.Address) (int64, error) {
 	return peerBalance.balance, nil
 }
 
+func (a *Accounting) Balances() (map[string]int64, error) {
```

Comment:
why do we need a function that wraps another function? you can just rename `getPeerBalances` to `Balances`